### ISSUE TITLE
Getting 'Response code 400 (Bad Request)' when trying to publish after uploading a new version

### ISSUE THREAD

**Issue by fwextensions** [DrewML/chrome-webstore-upload#25]
The call to `uploadExisting()` succeeds, so I think all the secrets and tokens are correct.  When I go to the Chrome Web Store developer dashboard, I see the newly uploaded package waiting there to be published.  But when I call `publish()`, I keep getting:

	  { message: 'Response code 400 (Bad Request)',
	  host: 'www.googleapis.com',
	  hostname: 'www.googleapis.com',
	  method: 'POST',
	  path: '/chromewebstore/v1.1/items/ddbinmabfeaibekibbpbnblmkddlkhpo/publish?publishTarget=default',
	  statusCode: 400,
	  statusMessage: 'Bad Request' }
	
Is there any way to debug this?

**Comment by fwextensions** [comment]
After reading through other closed issues where people were getting publishing errors, I went back to the store dashboard to try publishing from there, to see if there was something wrong with the uploaded package.  Turns out, extensions are now required to have at least one screenshot, and I was using a test extension that I'd created before that requirement, so publishing manually via the dashboard failed, too.  After adding a screenshot, publishing via the API works.  

Hopefully this helps someone trying to figure out Google's execrable API errors.
